src/metabase/pulse/render/style.clj

```diff
@@ -60,13 +60,9 @@
   "Color for dark text."
   "#4C5773")
 
-(def ^:const color-header-row-border
-  "Used as color for the bottom border of table headers for charts with `:table` vizualization."
-  "#EDF0F1")
-
-(def ^:const color-body-row-border
-  "Used as color for the bottom border of table body rows for charts with `:table` vizualization."
-  "#F0F0F04D")
+(def ^:const color-border
+  "Used as color for the border of table, table header, and table body rows for charts with `:table` vizualization."
+  "#F0F0F0")
```

Comment:
This uses `#F0F0F0` as specced in the design, and since the new design only one border-color it's safe to remove `color-header-row-border` too.